### DISCUSSION TITLE
No WebRAVE visibility for this TauDEM project

### DISCUSSION THREAD

**Discussion by MattReimer** [Riverscapes/riverscapes-website-jekyll#23] [Riverscapes Viewer] [answered]
@MattReimer and @philipbaileynar , we (@nick4rivers and @nbouwes ) were looking at this [TauDEM project](https://webrave.riverscapes.xyz/#/Anabranch/f003f545-4c93-49e4-8b30-69884e2ae2be) for Dollar Ridge and nothing displays in webRAVE:
![image](https://user-images.githubusercontent.com/9723905/142071119-ed9d4233-a367-4c49-92c7-f808cb17079b.png)

Although this is a 0.5 m resolution DEM, the whole project is only 1.6 GB and the DEM itself is only 121.6 MB.

It works fine in QRAVE: 

![image](https://user-images.githubusercontent.com/9723905/142071741-5fef218a-b803-47d7-85ec-0d065000adb0.png)

**Comment by MattReimer** [comment]
These layers got uploaded with windows paths. I'll talk to @KellyMWhitehead about it.

This can happen when using the `rscli` from a windows box 

there is a bug to prevent this:

- https://github.com/Riverscapes/rs-web-monorepo/issues/62

but likely what needs to happen is a quick python script needs to be written to fix all these paths and then re-queue the tiling.

**Comment by MattReimer** [comment]
Also I should mention I'm moving this into the `rs-web-monorepo` so it can live with the issues and fixes for this

Also, that project was run on Aug 4, 2021 4:36 PM (3 months ago) and we've since stopped uploading anything from windows boxes so we shouldn't see this problem going forward. This is mainly about a legacy fix for a project we'd rather not re-run

**Comment by MattReimer** [comment]
From @joewheaton:

> Good  stuff @MattReimer. I am trying to get folks that are consuming our projects in the habit of posting issues they find here and then @jtgilbert will keep an eye on them and mention them to the likes of you @philipbaileynar, @KellyMWhitehead or the @Riverscapes/riverscape-project-curators. If it is a question that warrants logging a bug or request, we (developers) can do that and track it there.  However, when resolved, please report back here so that the people finding the bug (i.e. @nbouwes in this example) get notified when it is fixed.

Roger that

> Just to clarify, on this one what I'm hearing is the problem had nothing to do with file size or LiDAR (note this is a 0.5 m resolution raster) and everything to do with windows paths (#62).  That's good.

Yup. Far as I know. All the `topo` projects are broken for the same reason right now.

**Comment by joewheaton** [comment, marked as answer]
@MattReimer you guys fixed this. The replacement project that works is https://webrave.riverscapes.xyz/#/Anabranch/f052b65d-dddc-4277-b0eb-ad27eda3c723. I am marking this as answered and requesting we delete the old broken project. 

See: https://github.com/Riverscapes/RiverscapesData/issues/194

FYI @sshahverdian, @nick4rivers  and @nbouwes
